### ISSUE TITLE
Installation of Mac OS El Capitan on Mac mini M2 Pro / crahses - HD Issue?

### ISSUE THREAD

**Issue by macraq2** [adespoton/utmconfigs#13]
Thanks for the excellent idea of providing setups for UTM here for much macOS versions. On UTM 4.16 I started several attempts yesterday to install a virtual machine with El Capitan on my Mac Mini M2 Pro (16/512) using the Mac OS 10.11 preset. The process runs flawlessly up to the point where the Mac OS is supposed to be installed on the hard disk. The error message appears that the Mac OS cannot be installed on this hard disk. There is also no drive of sufficient size visible in Disk Utility. I have tried several times, each time before a new installation attempt, to set up a sufficiently large hard disk in the preset. But this does not work for some unknown reason. Also in the virtual machine label - below machine and memory next to the hard disk icon a much too small size - mostly small than 1 MByte - is displayed. Probably this also leads to the fact that at the latest when trying to use the Apple Disk Utility the virtual machine crashes. What am I doing wrong?

![Bildschirm_foto 2023-03-23 um 13 28 01](https://user-images.githubusercontent.com/128677215/227205508-64cbdcc8-cb89-4589-bd99-6bf1e122ba05.png)

![Bildschirm_foto 2023-03-23 um 14 13 50](https://user-images.githubusercontent.com/128677215/227215414-07a9d2be-27e6-4cde-9d23-601832f4acba.png)

**Comment by adespoton** [comment]
That’s really odd… the config includes a pre-formatted HFS+ image of 20GB IIRC, already set up on the right IDE bus.These were set up under UTM 3 though - it’s possible something has happened during the conversion to UTM 4 format.  I’ll dig into it when I get some time.Meanwhile, you may need to remove the disk image and create a new 20GB (or larger) one on IDE, and then when the DVD image boots, go into Disk Utility and format the new drive as HFS+.  It should then be selectable in the installer.-Em On Mar 22, 2023, at 11:52 PM, macraq2 ***@***.***> wrote:﻿
Thanks for the excellent idea of providing setups for UTM here for much macOS versions. I started several attempts yesterday to install a virtual machine with El Capitan on my Mac Mini M2 Pro (16/512) using the Mac OS 10.11 preset. The process runs flawlessly up to the point where the Mac OS is supposed to be installed on the hard disk. The error message appears that the Mac OS cannot be installed on this hard disk. There is also no drive of sufficient size visible in Disk Utility. I have tried several times, each time before a new installation attempt, to set up a sufficiently large hard disk in the preset. But this does not work for some unknown reason. Also in the virtual machine label - below machine and memory next to the hard disk icon a much too small size - mostly small than 1 MByte - is displayed. Probably this also leads to the fact that at the latest when trying to use the Apple Disk Utility the virtual machine crashes. What am I doing wrong?

**Comment by macraq2** [comment]
Thank you for the quick tips!
1. enlarging the hard disk or deleting and recreating in the configurator did not lead to any improvement.
2. as a test I installed the last 3.x version (3.24) for Mac OS. Now I can use Disk Utility to edit, delete, partition the disks without crashing. However, when trying to install the system, it still gives the error message: "copy this application...." See image above.

**Comment by macraq2** [comment]
Unfortunately, that did not work either. I made a lot of attempts yesterday. But unfortunately no success.
1. the Installer.app conversion recommended in the program window does not work on the M2 Mac because I cannot install the OSX.DMG in the Applications folder.
2. so I used terminal commands to convert an ElCapitan.DMG (downloaded from Apple server) to an ISO file. But with this ISO no (third) choice for installation appears in the boot loader. Also an attempt with Yosemite (OSX 10.10) does not work.
3. the ISO file I had used before, which at least led up to Disk Utility (see above), I found here https://archive.org/details/macos-collection .
4. still hoping to find a solution. Otherwise, I'll have to go a completely different route, as I desperately need El Capitan on my new M2 Pro Mac for an old accounting program.

So as before I am very grateful for any helping tip.


Translated with www.DeepL.com/Translator (free version)

**Comment by adespoton** [comment]
Yeah; M1 Macs can’t do the conversion.  However, an emulated x86 Mac can :)Then the challenge becomes “how do I set up an emulated Mac to download and convert the installer to set up an emulated Mac…?”-Em On Mar 25, 2023, at 1:05 AM, macraq2 ***@***.***> wrote:﻿
Unfortunately, that did not work either. I made a lot of attempts yesterday. But unfortunately no success.

the Installer.app conversion recommended in the program window does not work on the M2 Mac because I cannot install the OSX.DMG in the Applications folder.
so I used terminal commands to convert an ElCapitan.DMG (downloaded from Apple server) to an ISO file. But with this ISO no (third) choice for installation appears in the boot loader. Also an attempt with Yosemite (OSX 10.10) does not work.
the ISO file I had used before, which at least led up to Disk Utility (see above), I found here https://archive.org/details/macos-collection .
still hoping to find a solution. Otherwise, I'll have to go a completely different route, as I desperately need El Capitan on my new M2 Pro Mac for an old accounting program.

So as before I am very grateful for any helping tip.
Translated with www.DeepL.com/Translator (free version)

**Comment by macraq2** [comment]
Thanks :-) After a long day now the success. I found out that my application runs from OSX 10.7 to 10.11. With Lion and UTM 3.24 I came to success. I extracted the InstallESD.dmg from the DMG as described and assigned this as the installation drive. In your config file I later only enabled file sharing on the network. Thanks for the prepared config files. The virtual machine has been running on the M2 for a few hours now. Great. I was about to give up today and switch to Parallels. Then I had to learn that on Apple Silicon only OS 12 and 13 are supported. Your work and UTM are really a stroke of luck.

**Comment by adespoton** [comment]
I should probably add that to the instructions; bootstrapping with 10.7 will indeed let you grab the installer for any of the later OS versions and either overinstall or create an iso.-Em On Mar 25, 2023, at 1:11 PM, macraq2 ***@***.***> wrote:﻿
Thanks :-) After a long day now the success. I found out that my application runs from OSX 10.7 to 10.11. With Lion and UTM 3.24 I came to success. I extracted the InstallESD.dmg from the DMG as described and assigned this as the installation drive. In your config file I later only enabled file sharing on the network. Thanks for the prepared config files. The virtual machine has been running on the M2 for a few hours now. Great. I was about to give up today and switch to Parallels. Then I had to learn that on Apple Silicon only OS 12 and 13 are supported. Your work and UTM are really a stroke of luck.
